x-pack/plugins/lens/public/indexpattern_datasource/field_item.tsx

```diff
@@ -198,22 +183,20 @@ export const InnerFieldItem = function InnerFieldItem(props: FieldItemProps) {
       ownFocus
       className="lnsFieldItem__popoverAnchor"
       display="block"
+      data-test-sub="lnsFieldListPanelField"
```

Comment:
🏳️ There is also a test targeting this DTS that I can't seem to fix

Reply:
It's just the `j` missing in `data-test-sub`

Reply:
Oh goodness... Haha. Thanks!